config/controlplane.cluster.x-k8s.io_kopscontrolplanes.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
Is this downgrade expected?

Reply:
yeah, good call, not sure if it's fake news though, i'll double check here